lms/djangoapps/teams/static/teams/js/views/team_utils.js

```diff
@@ -37,8 +38,37 @@
                         return membership.user.username === requestUsername;
                     })
                 );
+            },
+
+            /**
+             * Returns true if the discussion thread belonging to
+             * `team` is accessible to the user. This is the case
+             * if the user is privileged (i.e., a community TA,
+             * moderator, or administrator), or if the user
+             * belongs to the team.
+             */
+            readOnlyDiscussion: function (isMember) {
+                return !($('.teams-content').data('privileged') || isMember);
+            },
+
+            showMessage: function (message) {
+               $('.wrapper-msg').removeClass('is-hidden');
+               $('.msg-content .copy').text(message);
+               $('.wrapper-msg').focus();
+            },
+
+            /**
+             * Parse `data` and show user message. If parsing fails than show `genericErrorMessage`
+             */
+            parseAndShowMessage: function (data, genericErrorMessage) {
```

Comment:
This is coupling parsing and setting the message. It would be better if this only parsed and the caller could do `this.showMessage(parseMessage(data, defaultMessage))`.